package.json

```diff
@@ -109,6 +110,7 @@
         "onCommand:workbench.action.tasks.runTask",
         "onDebugInitialConfigurations",
         "onDebugResolve:docker",
+        "onFileSystem:docker",
```

Comment:
I had the same mixed feelings, especially since I could envision maybe a similar URI scheme for access to files in a (non-running) Docker image, but couldn't think of anything that didn't feel as ugly.  Since we use `docker` for our debug configuration type, I felt using a similar name was probably ok.  Open to suggestions, though.  (`vscode-docker` feels too coupled to our extension, rather than being a general URI scheme for accessing Docker container files.)